docs/source/concept_models/vivarium_csu_multiple_myeloma_2/concept_model.rst

```diff
@@ -168,7 +168,7 @@ United States.
 Population description:
 
 * Cohort type: Prospective closed cohort of individuals aged 15 years and older. The sim duration is 15 years (see below), so results above age 30 will not be impacted by the open/closed distinction; essentially all multiple myeloma occurs at age 30+.
-* Size of largest starting population: 100,000 simulants
+* Size of largest starting population: 2,000 simulants
```

Comment:
I think there is some sweet-spot trade off the the number of simulants and the overhead time for loading the data needed to set up the simulation, so maybe 2,000 simulants is too small.  Engineers might be able to advise.